### PR TITLE
(DOCSP-26989): @realm/reactify 'Index a Property'

### PR DESCRIPTION
**Pull request by davidhou17**
### Jira

- https://jira.mongodb.org/browse/DOCSP-26989

### Staged Changes

- [Index a Property](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-26989/sdk/react-native/realm-database/schemas/index/)

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by mongomoe — COMMENTED** [review]
The content of this PR look great, but I don't see the Book class in `examples/react-native/__tests__`. Did you forget to git add those files? 

We need a Book.ts in the `examples/react-native/__tests__/ts/models/` folder and a Book.js in the `examples/react-native/__tests__/js/models/` folder.

File: source/sdk/react-native/realm-database/schemas/index.txt [hunk 1]
```diff
@@ -15,26 +15,42 @@ Index a Property - React Native SDK
...
-      :emphasize-lines: 4
\ No newline at end of file
+.. tabs-realm-languages::
+
+    .. tab::
+       :tabid: typescript
+
+       .. literalinclude:: /examples/generated/react-native/ts/Book.snippet.ts-book-schema.ts
+          :language: typescript
+          :emphasize-lines: 8
```
Comment (mongomoe) [inline source/sdk/react-native/realm-database/schemas/index.txt:48]:
```suggestion
          :emphasize-lines: 8
          :linenos:
```
To add line numbers to the code snippet

File: source/sdk/react-native/realm-database/schemas/index.txt [hunk 2]
```diff
@@ -15,26 +15,42 @@ Index a Property - React Native SDK
...
+
+
+    .. tab::
+       :tabid: javascript
+
+       .. literalinclude:: /examples/generated/react-native/js/Book.snippet.js-book-schema.js
+          :language: javascript
+          :emphasize-lines: 5
```
Comment (mongomoe) [inline source/sdk/react-native/realm-database/schemas/index.txt:56]:
```suggestion
          :emphasize-lines: 5
          :linenos:
```

**Review by mongomoe — APPROVED** [review]
LGTM 👍

**Review by takameyer — APPROVED** [review]

**Pull request merged**